console-api/Cargo.toml

```diff
@@ -40,7 +40,7 @@ tracing-core = "0.1.17"
 futures-core = "0.3"
 
 [dev-dependencies]
-tonic-build = { version = "0.10", default-features = false, features = [
+tonic-build = { version = "0.11", default-features = false, features = [
```

Comment:
When updating `tonic-build`, the `console-api` interface needs to be updated. That makes this a breaking change for both `console-api` and `console-subscriber` which both include tonic in the public API.

Reply:
Amended the commit but
```
cargo run -p xtask -- gen-proto
```
didn't do anything and I'd be surprised it should because prost is still the same and the protos left untouched.

Reply:
OK, it may be that this change doesn't affect the output, but it is still a breaking change for `console-api` and `console-subscriber` and we'll have to treat it as such.

Reply:
As I said, I amended the commit with the breaking change trailer. Anything else that's missing?

Reply:
Sorry, I missed that. This is good to go. However, we've got another change which is going to bump the MSRV higher, I'd prefer to put that in first and then this one won't need the MSRV bump. Hopefully we'll get that other one in soon.